crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -396,7 +397,13 @@ define_tedge_config! {
                 #[tedge_config(example = "8001", default(value = 8001u16))]
                 port: u16,
             }
-        }
+        },
+
+        bridge: {
+            /// Set the bridge local clean session flag (required mosquitto >= 2.0.0)
+            #[tedge_config(example = "auto", default(value = false))]
+            include_localcleansession: AutoFlag,
+        },
```

Comment:
I added here the setting `c8y.bridge.include_localcleansession`
while the ticket refer `c8y.include_localcleansession`.

@reubenmiller any preferences ?

Reply:
I'm evaluating options now.

Reply:
What about `c8y.bridge.include.local_cleansession` as the mosquitto setting is called `local_cleansession`, and that way it is easier to understand what value it is referring to (and it will be easier to find in the code/configuration)